src/test/regress/expected/grant_role_2pc.out

```diff
@@ -0,0 +1,159 @@
+CREATE SCHEMA grant_role2pc;
+SET search_path TO grant_role2pc;
+set citus.enable_create_database_propagation to on;
+CREATE DATABASE grant_role2pc_db;
+\c grant_role2pc_db
+SHOW citus.main_db;
+ citus.main_db
+---------------------------------------------------------------------
+ regression
+(1 row)
+
+SET citus.superuser TO 'postgres';
+CREATE USER grant_role2pc_user1;
+CREATE USER grant_role2pc_user2;
+CREATE USER grant_role2pc_user3;
+CREATE USER grant_role2pc_user4;
+CREATE USER grant_role2pc_user5;
+CREATE USER grant_role2pc_user6;
+CREATE USER grant_role2pc_user7;
+\c grant_role2pc_db
+--test with empty superuser
+SET citus.superuser TO '';
+grant grant_role2pc_user1 to grant_role2pc_user2;
+ERROR:  No superuser role is given for Citus main database connection
+HINT:  Set citus.superuser to a superuser role name
+SET citus.superuser TO 'postgres';
+grant grant_role2pc_user1 to grant_role2pc_user2 with admin option granted by CURRENT_USER;
+\c regression
+select result FROM run_command_on_all_nodes(
+    $$
+    SELECT array_to_json(array_agg(row_to_json(t)))
+    FROM (
+    SELECT member::regrole, roleid::regrole as role, grantor::regrole, admin_option
+    FROM pg_auth_members
+    WHERE member::regrole::text = 'grant_role2pc_user2'
+    order by member::regrole::text
```

Comment:
To prevent test outputs getting flaky, can we add `roleid::regrole::text` into all such order by clauses in this test file?

```suggestion
    order by member::regrole::text, roleid::regrole::text
```

Reply:
Fixed in all queries